_data-prepper/peer-forwarder.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: default
-title: Peer Forwarder
+title: Peer forwarder
 nav_order: 12
 ---
 
 # Peer Forwarder
 
-Peer Forwarder is an HTTP service that performs peer forwarding of an `event` between Data Prepper nodes for aggregation. This HTTP service uses a hash-ring approach to aggregate events and determine which Data Prepper node it should handle on a given trace before rerouting it to that node. Currently, Peer Forwarder is supported by the `aggregate`, `service_map_stateful`, and `otel_trace_raw` [processors]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/processors/).
+Peer forwarder is an HTTP service that performs peer forwarding of an `event` between Data Prepper nodes for aggregation. This HTTP service uses a hash-ring approach to aggregate events and determine which Data Prepper node it should handle on a given trace before rerouting it to that node. Currently, Peer forwarder is supported by the `aggregate`, `service_map_stateful`, and `otel_trace_raw` [processors]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/processors/).
```

Comment:
These changes abandon the practice of using initial caps for Peer Forwarder. But the feature now appears two ways in the text here:
1. Peer forwarder (example, here, line 9)
2. peer forwarder (example, all table descriptions and line 112)
I guess it's easiest to get rid of capitalization for both words. So, unless it starts a sentence, using lower case for "peer forwarder" seems the way to go. Does that sound right?